docs/deployment/git.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 1
+---
+
+# Git
+
+Before you deploy please make sure you have uploaded your code to a git repository. This is required for the deployment on Vercel.
+
+## GitHub
+
+GitHub is a code hosting platform for version control and collaboration. It lets you and others work together on projects from anywhere.
+
+[Github](https://github.com/join)
+
+## GitLab
```

Comment:
Recommend leaving out gitlab/bitbucket - it'll only confuse non-technical people

Reply:
ye ig, didnt think of that